scripts/lib/CIME/XML/generic_xml.py

```diff
@@ -250,6 +250,13 @@ def get_children(self, name=None, attributes=None, root=None):
 
         return children
 
```

Comment:
We do not need this additional function - use get_optional_child or get_children instead.

Reply:
This return is from function get_children. I don't understand why you want to delete it. Please clarify.